src/SFA.DAS.Assessor.Domain/Entities/Ofs/OfsOrganisation.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace SFA.DAS.Assessor.Functions.Domain.Entities.Ofs
+{
+    public record class OfsOrganisation
+    {
+        public int Ukprn { get; init; }
+        public string RegistrationStatus { get; init; }
+        public string HighestLevelOfDegreeAwardingPowers { get; init; }
+
+        public OfsOrganisation(int ukprn, string registrationStatus, string highestLevelOfDegreeAwardingPowers)
+        {
+            Ukprn = ukprn;
+            RegistrationStatus = registrationStatus;
+            HighestLevelOfDegreeAwardingPowers = highestLevelOfDegreeAwardingPowers;
+        }
+    }
```

Comment:
Here we can take advantage of the [positional record syntax](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record), which does all of this in one line:
```suggestion
    public record class OfsOrganisation(int Ukprn, string RegistrationStatus, string HighestLevelOfDegreeAwardingPowers)
```

Reply:
Done for Ofs and the Ofqual ones